docs/04-operation-guides/operations/01-install-kyma.md

```diff
@@ -6,7 +6,23 @@ You can simply use the default Kyma installation, or modify it as it fits your p
 
 ## Default installation
 
-If you use the `deploy` command without any flags, Kyma provides a default domain. 
+Meet the prerequisites, provision a k3d cluster, and use the `deploy` command to run Kyma locally.
+
+### Prerequisites
+
+- [Kyma CLI](https://github.com/kyma-project/cli)
+- [Docker](https://docs.docker.com/get-docker/)
+- [k3d](https://k3d.io/#installation)
+
+### Provision and install
+
+You can either use an out-of-the-box k3d cluster or choose any other cluster provider. To quickly provision a k3d cluster run:
+
+  ```bash
+  kyma provision k3s
+  ```
+
+If you use the `deploy` command without any flags, Kyma provides a default domain.
 For example, if you install Kyma on a local cluster, the default URL is `https://console.local.kyma.dev`.
```

Comment:
We need to remove console from description. It is not part of Kyma 2.0. The console can be started locally. @valentinvieriu should provide the best option (docker / npx command).

Reply:
👍   I'm removing that part and creating a follow-up [issue](https://github.com/kyma-project/kyma/issues/11688) to add the details once we have them.

Reply:
👍  I'm removing the line about the Console and creating a [follow-up issue](https://github.com/kyma-project/kyma/issues/11688). TWs will update the doc once we have the required information. @valentinvieriu, I sent you a message on Slack regarding this - we're in touch.